membership_account_balance/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
ignore warning (non blocking).
Deprecated syntax might not be a problem: check if this is blocking.

Anyway I fear the travis for the repo v8.0 might have many other issues.